frontend/packages/ux-editor/src/components/config/editModal/EditOptions.tsx

```diff
@@ -56,19 +64,19 @@ export function EditOptions({
   }, [editFormId, initialSelectedOptionType]);
 
   const handleOptionsTypeChange = (value: SelectedOptionsType) => {
```

Comment:
Perhaps this is a preference, but could we write the code in a more DRY manner like this?

```typescript
const handleOptionsTypeChange = (selectedOption: SelectedOptionsType): void => {
   const newOptionsType = selectedOption === SelectedOptionsType.CodeList ? SelectedOptionsType.Manual : SelectedOptionsType.CodeList;
   const optionsToUpdate = newOptionsType === SelectedOptionsType.Manual ? { options: [] } : { optionsId: '' };

   setSelectedOptionsType(newOptionsType);
   delete component[newOptionsType === SelectedOptionsType.Manual ? 'optionsId' : 'options'];

   handleComponentChange({
     ...component,
     ...optionsToUpdate
   });
}
```